src/api.ts

```diff
@@ -9,7 +9,7 @@ function wait(delay: number) {
 
 export async function getPeople(): Promise<Person[]> {
   // keep this delay for testing purpose
-  return wait(500)
+  return wait(1000)
```

Comment:
Why did you change the delay?

Reply:
oh, sorry, it was mistake from my side, I forgot to edit it in the final version